ipfs-cluster-service/main.go

```diff
@@ -261,7 +261,82 @@ removal, upgrade state using this command, and restart every peer.
 					Action: func(c *cli.Context) error {
 						err := upgrade()
 						checkErr("upgrading state", err)
-						return err
+						return nil
+					},
+				},
+				{
+					Name:  "export",
+					Usage: "save the IPFS Cluster state to a human readable json file",
+					Description: `
+
+This command reads the current cluster state and saves it as json for 
+human readability and editing.  In general any version of cluster will only
+work with the latest state format, so out of date snapshots will be migrated
+before the export.
+`,
+					Action: func(c *cli.Context) error {
+						err := export()
```

Comment:
Not sure this can happen while cluster is running. maybe since it reads the last snapshot. We might refuse to run while cluster runs anyway.